### PR TITLE
Deprecate Aurora Editor Tabbar Style

### LINKED ISSUE #749: Depricate: Aurora Style Tabs
### Is your feature request related to a problem? Please describe.

Change

### Describe the solution you'd like

Deprecatie: Aurora Style Tabbar/Tabs

### Describe alternatives you've considered

None.

### Additional context

To make the tabbar code less complex.
If we try to debug the code now, the compiler is complaining about not being able to type check within a reasonable time.

### PR DESCRIPTION
**Pull request by 0xWDG**
## Summary

Deprecate Aurora Editor Tabbar Style

## Changes Made

Removed Aurora Editor Tabbar Style

## TODO

Remove grey background color of tab https://github.com/AuroraEditor/AuroraEditor/issues/755

## Motivation

None

## Testing

None

## Screenshots/GIFs

image

## Checklist

Please ensure all of the following are completed before submitting the PR:

- [x] Code has been tested and verified.
- [x] Documentation has been updated to reflect the changes.
- [x] All tests pass successfully.
- [x] Code follows the project's coding guidelines and style.
- [x] The branch is up-to-date with the latest changes from the main branch.
- [x] Reviewed by at least one other contributor.

## Related Issues

This PR addresses the following issue(s): #749

## Additional Notes

None

### CODE REVIEW

**Comment by aurora-care-bear** [comment]
Thanks for submitting a pull request!
We will review it as soon as possible.

File: AuroraEditor/Core/TabBar/UI/TabBar/TabBar.swift [hunk 1]
```diff
@@ -234,40 +234,22 @@ struct TabBar: View {
...
-                    }
-                }
             }
             // Tab bar tools (e.g. split view).
             trailingAccessories
         }
         .frame(height: TabBar.height)
-        .overlay(alignment: .top) {
-            // When tab bar style is `xcode`, we put the top divider as an overlay.
-            if prefs.preferences.general.tabBarStyle == .xcode {
```
Comment (nanashili) [inline AuroraEditor/Core/TabBar/UI/TabBar/TabBar.swift:249]: Wouldn't this need to stay tho?

Comment (0xWDG) [reply AuroraEditor/Core/TabBar/UI/TabBar/TabBar.swift:249]: nope, the code within the if contains a if for not Xcode 😅

Comment (nanashili) [reply AuroraEditor/Core/TabBar/UI/TabBar/TabBar.swift:249]: Okay cool

**Review by nanashili — APPROVED** [review]

**Comment by 0xWDG** [comment]
@aurora-care-bear please approve

**Review by aurora-care-bear — APPROVED** [review]
Approved on behalf of @0xWDG

**Comment by aurora-care-bear** [comment]
Hey @0xWDG,

Thanks for your contribution.
Your PR has been merged!
Your name will appear on https://auroraeditor.com/#contributors in a few moments, if you have contributed to the editor.

Thanks again for your contribution!

Kind Regards,
Aurora Editor Team.

**Pull request merged**